### ISSUE TITLE
[Feature request] Make tmate notify when someone joins the session

### ISSUE THREAD

**Issue by TheEvilSkeleton** [tmate-io/tmate#227]
The title says all. Can there be a feature where the host gets notified (visually and by sound) when a guest joins?

**Comment by nviennot** [comment]
Yes. You can use webhooks: https://github.com/tmate-io/tmate/wiki/Webhooks

**Comment by nicopace** [comment]
If you are attached to the session, you get a message in the notification area when someone joins.
you can also see it if you missed it by calling `tmux show-messages`
